modules/crowdfundings/lib/Mail.js

```diff
@@ -0,0 +1,70 @@
+const { createMail } = require('@orbiting/backend-modules-mail')
+const logger = console
+const {
+  MAILCHIMP_INTEREST_PLEDGE,
+  MAILCHIMP_INTEREST_MEMBER,
+  MAILCHIMP_INTEREST_MEMBER_BENEFACTOR,
+  MAILCHIMP_INTEREST_NEWSLETTER_DAILY,
+  MAILCHIMP_INTEREST_NEWSLETTER_WEEKLY,
+  MAILCHIMP_INTEREST_NEWSLETTER_PROJECTR
+} = process.env
+
+const mail = createMail([
+  { name: 'DAILY', interestId: MAILCHIMP_INTEREST_NEWSLETTER_DAILY, roles: ['member'] },
+  { name: 'WEEKLY', interestId: MAILCHIMP_INTEREST_NEWSLETTER_WEEKLY, roles: ['member'] },
+  { name: 'PROJECTR', interestId: MAILCHIMP_INTEREST_NEWSLETTER_PROJECTR, roles: [] }
+])
+
+mail.enforceSubscriptions = async ({ userId, hasJustPaid, isNew, pgdb, ...rest }) => {
```

Comment:
`userId, hasJustPaid, isNew, pgdb` are not in `rest`, which you assume here: on line 67 https://github.com/orbiting/republik-backend/pull/55/files#diff-23b403e7de5441412228029c50a4a833R67

Reply:
that's totally okay